src/main/java/org/cactoos/map/MapEnvelope.java

```diff
@@ -122,11 +125,14 @@ public final void clear() {
 
     @Override
     public final String toString() {
-        return new StringBuilder()
-            .append('{')
-            .append(new TextOf(this.entrySet()).toString())
-            .append('}')
-            .toString();
+        return new Concatenated(
+            new TextOf("{"),
+            new Joined(
+                ", ",
+                new Mapped<>(Object::toString, this.entrySet())
+            ),
+            new TextOf("}")
+        ).toString();
```

Comment:
I also want this method to be tested? Should I open an issue or can I add it right in this PR?

Reply:
@DmitryBarskov yes feel free to add test directly here, tests are always welcome :)